### DISCUSSION TITLE
Is there any way to change the mouse cursor on hover?

### DISCUSSION THREAD

**Discussion by jerluc** [FelixKratz/SketchyBar#313] [Q&A]
In particular, when moving your mouse over an item that has a `click_script`, I'd like to change the mouse cursor from the default to the pointer/hand cursor instead. Has anyone been able to do this? FWIW I'm also considering trying to figure out where in the code I could look to try to implement this as a new feature/config option.

**Comment by FelixKratz** [comment]
Thats a nice idea, it could be implemented here:
https://github.com/FelixKratz/SketchyBar/blob/48709c233c4015f9a593c818a1f5e6ca77f4ec24/src/bar_item.c#L215-L226

Thats the latest point in the code reached by all mouse entered and mouse exited events for a specific item.

Wether an item can be clicked or not is encoded in the `update_mask` of the item, i.e.:
```c
bool clickable = (bar_item->update_mask & UPDATE_MOUSE_CLICKED)
                 || (bar_item->click_script && strlen(bar_item->click_script) > 0);
```
